packages/core/test/test-util/test-services/v4/multiple-schemas-service/TestEntity3.ts

```diff
@@ -13,6 +13,7 @@ import {
   EdmTypeField,
   EntityBuilderType,
   EntityV4,
+  EnumFieldV2,
```

Comment:
As I mentioned the name with `V2` strongly triggers oData `V2` like here EntityV4 is for oData version. So I would prefer to have a different extension.